### PR TITLE
added .icns file 

### PR DESCRIPTION
**Pull request by pressqu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "files": [
       "build/**/*",
       "main/dist.js",
-      "!node_modules/**/*"
+      "!node_modules/**/*",
+      "./assets/*"
```
Comment (Xiphe) [inline package.json:26]: I currently also store the source files of the icon and other stuff in the assets folder and would not want to have them added to the bundle. I'd rather whitelist the exact assets that should be added.

**Comment by Xiphe** [comment]
Hey @pressqu thanks for the contribution! Could you briefly describe what the additional `.icns` file is doing?

**Comment by pressqu** [comment]
Sure, when building the Mac app, it uses the BB icon for the budgetbudget.app

**Comment by Xiphe** [comment]
Hm. I was following this guide: https://www.electron.build/icons and added the [`icon.png`](https://github.com/Xiphe/budgetbudget/blob/alpha/assets/icon.png) for exactly that and it "works on my machine" ™.

Could you check if when you [download the latest release](https://github.com/Xiphe/budgetbudget/releases/download/v0.2.0-alpha.2/BudgetBudget.dmg) the icon is correct?

**Pull request closed without merging**